### DISCUSSION TITLE
希望评论系统可以支持Artalk

### DISCUSSION THREAD

**Discussion by dianso** [gaowei-space/markdown-blog#40] [General]
目前的评论系统依赖github，希望支持这个go写的artalk，主要是支持自己部署，数据可以存到sqlite或者mysql等各种数据库。

github.com/ArtalkJS/Artalk

作者你要是开放一个接口，自定义js和css，就支持各种评论系统了。

**Comment by gaowei-space** [comment]
我康康

**Reply by dianso** [reply]
> 我康康

求添加自定义JS CSS功能，我自己加到了footer.html结果弄的运行不起来了

**Reply by dianso** [reply]
> 我康康

作者估计你也忙。

我并不懂go语言，今天借助chatgpt的帮助，我还是成功将artalk和一些其他功能加进来了。

由于现在的go是内嵌了资源文件，我想修改layout.html文件添加js和css只能部署go开发环境，修改完后编译，然后docker打包，推送到我的仓库里，然后部署。整个过程耗时不多，就是麻烦了点。

希望作者可以提供自定义css和js功能，方案可以是传递参数，也可以默认在layout.html放置2个空的js和css文件，只需要修改这两个文件，重启容器就能生效。

或者就是不打包单文件，将web目录爆露出来，然后Dockerfile里面只定个有bash的系统，比如debian，这样容器最大也不过80M。

可以通过ssh到容器来修改layout.html文件。

**Comment by dianso** [comment]
我今天下载了0.1.1版本，这一版之后编译的都是单文件，资源文件static无法修改。

建议下个版本还把web目录暴露出来，这样方便修改html文件来实现一些功能。

也能实现自定义主题。

**Comment by gaowei-space** [comment]
get, 最近在忙其他的，等我找时间搞搞
